src/main/liberty/config/server.xml

```diff
@@ -11,5 +11,8 @@
 	<httpEndpoint host="*" httpPort="${default.http.port}" httpsPort="${default.https.port}" id="defaultHttpEndpoint"/>
 
 	<webApplication location="CloudantSample.war" contextRoot="/"/>
+	<variable name="cloudant.apikey" value="${API_KEY}"/>
```

Comment:
```suggestion
```
It may be a good idea to do a one-off test with an IBM Cloud hosted Cloudant, but for the sample this will likely need to use username and password